tests/unit/users.spec.ts

```diff
@@ -0,0 +1,203 @@
+import { expect, should } from 'chai';
+import * as fs from '../../src/api/fs';
+import * as auth from '../../src/api/auth';
+import { getConfig } from '../../src/api/utils';
+import { resolve } from 'path';
+
+let testPath = resolve(__dirname, 'test.json');
+let testAuth = {
+  organizations: [
+    {
+      name: 'bleenco',
+      teams: [ { name: 'developers', members: [ 'admin' ] } ],
+      members: [ { name: 'admin', role: 'owner' } ],
+      packages: []
+    }
+  ],
+  users: [
+    { name: 'admin', password: 12345, fullName: '' }
+  ],
+  packages: []
+};
+let testConfig = getConfig();
+
+describe('User roles, affiliation and permission specific tests', () => {
+
+  beforeEach(() => {
+    testAuth = {
```

Comment:
why do you re-declare `testAuth` object here, it is declared already before `describe`.